### PR TITLE
chore: remove plumbum

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

**Review by kszucs — APPROVED** [review]

File: ci/datamgr.py [hunk 1]
```diff
@@ -316,47 +316,32 @@ def cli(quiet):
...
+    '--directory',
+    type=click.Path(file_okay=False, dir_okay=True, path_type=str),
+    default=DATA_DIR,
+    help="Output directory",
+    show_default=True,
+)
+def download(repo_url, rev, directory):
+    shutil.rmtree(directory, ignore_errors=True)
+    subprocess.run(["git", "clone", repo_url, directory, "--depth", "1"])
+    subprocess.run(["git", "reset", "--hard", rev], cwd=directory)
```
Comment (kszucs) [inline ci/datamgr.py:344]: Actually this won't work for earlier revisions due to `--depth=1`.

I suggest to remove the depth argument If rev is defined.

**Review by kszucs — CHANGES_REQUESTED** [review]

File: ci/datamgr.py (hunk 1 above)
Comment (cpcloud) [reply ci/datamgr.py:344]: Yep, good catch. On it.

**Review by kszucs — APPROVED** [review]

**Comment by kszucs** [comment]
@cpcloud needs a rebase.

**Pull request merged**